dali/operators/image/peek_shape/peek_image_shape.h

```diff
@@ -74,20 +75,22 @@ class PeekImageShape : public Operator<CPUBackend> {
     const auto &input = ws.template Input<CPUBackend>(0);
     auto &output = ws.template Output<CPUBackend>(0);
     size_t batch_size = input.num_samples();
+    DALI_ENFORCE(input.type() == DALI_UINT8, "Input must be stored as uint8 data.");
 
     for (size_t sample_id = 0; sample_id < batch_size; ++sample_id) {
       thread_pool.AddWork([sample_id, &input, &output, this] (int tid) {
         const auto& image = input[sample_id];
         // Verify input
-        DALI_ENFORCE(image.ndim() == 1,
+        // TODO(klecki): Move the checks to scope above
+        DALI_ENFORCE(image.shape().sample_dim() == 1,
                       "Input must be 1D encoded jpeg string.");
-        DALI_ENFORCE(IsType<uint8>(image.type()),
-                      "Input must be stored as uint8 data.");
-        auto img = ImageFactory::CreateImage(image.data<uint8>(), image.size(), {});
+        DALI_ENFORCE(image.type() == DALI_UINT8, "Input must be stored as uint8 data.");
```

Comment:
While you're at it:
```suggestion
        DALI_ENFORCE(image.type() == DALI_UINT8, "The input must be raw, undecoded files stored as a flat uint8 array.");
```

Reply:
I hope this won't be the line that makes it 1k+